src/tools/cephfs/first-damage.py

```diff
@@ -60,14 +60,16 @@
 REPAIR_NOSNAP = None
 
 CEPH_NOSNAP = 0xfffffffe # int32 -2
+ROOT_INODE  = "1.00000000"
+LOST_FOUND_INODE  = "4.00000000"
 
 DIR_PATTERN = re.compile(r'[0-9a-fA-F]{8,}\.[0-9a-fA-F]+')
 
 CACHE = set()
 
 def traverse(MEMO, ioctx):
     for o in ioctx.list_objects():
-        if not DIR_PATTERN.fullmatch(o.key):
+        if not DIR_PATTERN.fullmatch(o.key) and o.key not in [ROOT_INODE, LOST_FOUND_INODE]:
```

Comment:
I'd say let's use tuple instead of list here

Reply:
why?

Reply:
no updates to the list as such, only purpose is reading, tuple is better performant, memory efficient than lists.

Reply:
> tuple is better performant, memory efficient than lists.

for just a couple of items?